hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/handlers/ConsensusUpdateTopicHandler.java

```diff
@@ -127,22 +130,22 @@ private boolean onlyExtendsExpiry(@NonNull final ConsensusUpdateTopicTransaction
     public void handle(@NonNull final HandleContext handleContext) {
         requireNonNull(handleContext);
 
-        final var topicUpdate = handleContext.body().consensusUpdateTopic();
+        final var txn = handleContext.body();
+        final var op = txn.consensusUpdateTopicOrThrow();
+
         final var topicStore = handleContext.writableStore(WritableTopicStore.class);
-        final var maybeTopic = requireNonNull(topicStore).get(topicUpdate.topicIDOrElse(TopicID.DEFAULT));
-        validateTrue(maybeTopic.isPresent(), INVALID_TOPIC_ID);
-        final var topic = maybeTopic.get();
-        validateFalse(topic.deleted(), INVALID_TOPIC_ID);
+        final var topic = topicStore.getTopic(op.topicIDOrElse(TopicID.DEFAULT));
```

Comment:
Do we need to check topic is not null here ?

Reply:
The same reason above.

Reply:
Agreed. Thank you!